javaharness/java/arcs/android/ArcsService.java

```diff
@@ -22,12 +25,23 @@
   @Inject
   ArcsShellApi arcsShellApi;
 
+  @Inject
+  AndroidArcsEnvironment environment;
```

Comment:
This new field can be avoided if we move the logic to ArcsShellApi.

Reply:
done